### ISSUE TITLE
Commands to reproduce the results

### ISSUE THREAD

**Issue by IssacCyj** [CVIR/TCL#1]
Thanks for this great repo!

I notice in the paper it says there're several stages in the training procedure in section3.3 (Pretraining→TCL →Finetuning). In section 3.2.1 it also mentioned using a supervised model to initialize weights. Do you mind sharing the training command for each stage in order to reproduce the results in the paper?

Thanks

**Comment by IssacCyj** [comment]
To clarify a little bit. I can see that in the code you have [supervised stage + TCL + Finetuning], but not pre-training stage using instance contrastive loss. Is the command provided in **Sample Code to train TCL** section in README the command to reproduce the results?

**Comment by Griffintaur** [comment]
**Sample Code to train TCL** section contains the command to execute results for  [Supervised stage + TCL+ Finetuning].
To use pretraining, we just initialized the weights of the network with the pre-trained weights of the model which was trained separately using instance contrastive loss.
